src/main/java/io/strimzi/kafka/bridge/http/RequestIdentifier.java

```diff
@@ -0,0 +1,86 @@
+ */
+public class RequestIdentifier {
+
+    static RequestType getRequestType(HttpServerRequest request) {
+
+        String requestPath = request.path();
+
+        //request path always starts with "/" which is not required in path parameters
+        String [] params = requestPath.substring(1).split("/");
+
+        //produce records
+        //request type = POST
+        //path = topic/{topic-name}
+        //spliting this path will return an array of length 2
+        //param[0] = "topic", param[1] = {topic-name}
+        if (request.method() == HttpMethod.POST && params.length == 2 && params[0].equals("topic")){
+            return RequestType.PRODUCE;
+        }
+
+        //all consumer enpoints starts with "/consumers"
+         else if (params[0].equalsIgnoreCase("consumers")) {
+
+            //consumer creation
+            //request type = POST
+            //path = consumers/{consumer-group}
+            //spliting this path will return an array of length 2.
+            //param[0] = "consumers", param[1] = {consumer-group}
+            if (request.method() == HttpMethod.POST && params.length == 2) {
+                return RequestType.CREATE;
+
+            }
+
+            //consumer subscription
+            //request type = POST
+            //path = consumers/{consumer-group}/instances/{instance-id)/subscription
+            //spliting this path will return an array of length 5
+            //param[0] = "consumers", param[1] = {consumer-group}, param[2] = "instances", param[3] = {instance-id}, param[4] = "subscription"
+            else  if (request.method() == HttpMethod.POST && params.length == 5 && params[params.length - 1].equals("subscription")) {
+                return RequestType.SUBSCRIBE;
+            }
+
+            //consume records
+            //request type = GET
+            //path = consumers/{consumer-group}/instances/{instance-id)/records
+            //spliting this path will return an array of length 5
+            //param[0] = "consumers", param[1] = {consumer-group}, param[2] = "instances", param[3] = {instance-id}, param[4] = "records"
+            else if (request.method() == HttpMethod.GET && params.length == 5 && params[params.length - 1].equals("records")){
+                return RequestType.CONSUME;
+            }
+
+            //commit offsets
+            //request type = POST
+            //path = consumers/{consumer-group}/instances/{instance-id)/offsets
+            //spliting this path will return an array of length 5
+            //param[0] = "consumers", param[1] = {consumer-group}, param[2] = "instances", param[3] = {instance-id}, param[4] = "offsets"
+            else if (request.method() == HttpMethod.POST && params.length == 5 && params[params.length - 1].equals("offsets")){
+                return RequestType.OFFSETS;
+            }
+        }
+
+        return RequestType.INVALID;
+    }
+}
```

Comment:
Maybe a better formatting, a switch on the `request,method()` and then for each case the checks on params ? wdyt ?